test/docker-compose.yml

```diff
@@ -1,9 +1,7 @@
-version: '3.4'
-
 services:
   nats:
-    image: nats:2.8.4
-    command: --js # enable JetStream
+    image: nats:2.10.18
```

Comment:
This image is minimal, pretty much contains just `nats`, this means that the healthcheck doesn't work. I suggest using the alpine version, and even that one contains only `wget` (no `curl`).

Here's how I solved it in the pubsub connector - https://github.com/conduitio-labs/conduit-connector-nats-pubsub/blob/f8d22137798775c07e49ad18524536b5e5433c43/test/docker-compose.yml#L3-L11

Reply:
Fixed in https://github.com/conduitio-labs/conduit-connector-nats-jetstream/pull/137/commits/3a24eabc786a7c66923efcee1fac69a2a592c73b.